Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -251,7 +256,8 @@ void MemoryWidget::ConnectWidgets()
   connect(m_find_ascii, &QRadioButton::toggled, this, &MemoryWidget::ValidateSearchValue);
   connect(m_find_hex, &QRadioButton::toggled, this, &MemoryWidget::ValidateSearchValue);
 
-  connect(m_set_value, &QPushButton::clicked, this, &MemoryWidget::OnSetValue);
+  connect(m_from_file, &QPushButton::clicked, this, [this] { MemoryWidget::OnSetValue(true); });
```

Comment:
I think it would be better for this new button to have its own method rather than trying to implement it in the existing one.